topics/go/algorithms/data/hash/doc.go

```diff
@@ -0,0 +1,31 @@
+package hash
```

Comment:
If you put this last, immediately under the comment, the godocs will use this information.

Reply:
Done